scripts/issues.coffee

```diff
@@ -0,0 +1,40 @@
+# Description:
+#   Create GitHub issues for CH beheer and PC.COM. Please be concise.
+#
+# Dependencies:
+#   octonode
+#
+# Configuration:
+#   HUBOT_GITHUB_TOKEN = <accessToken>
+#
+# Commands:
+#   @pccom <query> - Creates a new issue and tags it 'pccom'
+#   @beheer <query> - Creates a new issue and tags it 'beheer'
+
+github = require 'octonode'
+
+githubToken = process.env.HUBOT_GITHUB_TOKEN;
+
+module.exports = (robot) ->
+  unless githubToken?
+    robot.logger.error "Missing HUBOT_GITHUB_TOKEN in environment"
+    return
+  robot.listen(
+    (message) ->
+      console.log message
+      match = message.text.match /^@(pccom|beheer) (?:me )?(.*)/i
+      if match
+        data = {
+          title: match[2],
+          body: "Issue added by #{message.user.name}"
+          labels: ['slack', match[1]]
+        }
+        client = github.client githubToken
+        client.repo("WISVCH/pccom-beheer").issue data, (err) ->
+          if err
+            throw new Error "Github Error: #{err.message}"
```

Comment:
De github issue code naar de `(response)` verplaatsen en dan pas `response.reply` doen als er daadwerkelijk een issue is aangemaakt?